workflow/schemas/config.schema.yaml

```
@@ -22,6 +22,20 @@ properties:
+      build:
+        type: string
+      release:
+        type: string
+  allOf:
+    - if:
+        properties:
+          ensembl_species:
+            "type": string,
+            "min_length": 1
+      then:
+        required: ["build","release"]
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)